core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -70,14 +70,24 @@
     </li>
 
     {% if username %}
-      <li ng-if="isLargeScreen" class="dropdown" popover-is-open="postTutorialHelpPopoverIsShown"
+    <div class="oppia-help-dropdown nav navbar-nav oppia-navbar-nav pull-right ng-cloak">
```

Comment:
However, you do want to move `oppia-help-dropdown` to the right of `oppia-navbar-nav`